connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -137,12 +169,12 @@ public class ConnectorConfig extends AbstractConfig {
     public static final String CONFIG_RELOAD_ACTION_CONFIG = "config.action.reload";
     private static final String CONFIG_RELOAD_ACTION_DOC =
             "The action that Connect should take on the connector when changes in external " +
-            "configuration providers result in a change in the connector's configuration properties. " +
```

Comment:
nit: There's a bunch of unrelated changes to indenting and such in ConnectorConfig; Are these intentional?

They have the potential to cause merge conflicts with later PRs, or maybe they leaked in?